### PR TITLE
HW-345: Fix peoples tab page design

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**
people-tab-issues

**After**
resolved-people-tab-issues

**Shoreditch PR:** [#29](https://github.com/civicrm/org.civicrm.shoreditch/pull/29)

**Note:**
- Discussed with Alessandro to add all civicase changes on civicase repo and only genric chnages should go on Shoreditch repo.
- Confirmed with Jon about "Tag buttons", "Radio buttons" and "Check box". It should styled same as invision design.

### CODE REVIEW

File: ang/civicase/ViewPeople.html [hunk 1]
```diff
@@ -31,15 +31,15 @@
       </select>
     </div>
   </div>
-  <div class="pull-right">
+  <div class="pull-right client-role">
```
Comment (colemanw) [inline ang/civicase/ViewPeople.html]: Let's prefix this class with "case" like the others.

File: css/caseView.css [hunk 2]
```diff
@@ -1,30 +1,74 @@
 /* CSS rules for Angular civicase view */
+#branding h1.page-title {
+  margin: 0 0 20px !important;
+}
```
Comment (colemanw) [inline css/caseView.css]: `#branding` is part of a particular Drupal theme. So I don't think this style rule is going to work.

Comment (colemanw) [inline css/caseView.css]: I've just pushed a PR https://github.com/civicrm/civicrm-core/pull/10711 which will allow you to target it like

    h1.crm-page-title

File: ang/civicase/ViewPeople.html (hunk 1 above)
Comment (mukeshcompucorp) [inline ang/civicase/ViewPeople.html]: Fixed.

File: css/caseView.css (hunk 2 above)
Comment (mukeshcompucorp) [inline css/caseView.css]: Yes, you are right `#branding` is coming from Drupal theme but there is some CSS coming from the theme and I need to override the margin property. That's why I had used `#branding` If we will not use `#branding` then we can't override the theme css. Please let me know if you have any other solution.

Comment (colemanw) [inline css/caseView.css]: This sounds like a problem to be addressed in the shoreditch theme since it would affect more than just CiviCase.

Comment (mukeshcompucorp) [inline css/caseView.css]: But the `#branding` is coming from Drupal theme not from shoreditch theme. Do you understand my issue exactly ?

Comment (colemanw) [inline css/caseView.css]: Yes I understand that you wish to alter the page title styling. But if all of CiviCRM (or CiviHR) is to have a consistent look & feel, then we would want the style rule to be in the shoreditch theme, not just the CiviCase component.

Comment (mukeshcompucorp) [inline css/caseView.css]: Fixed.

**Pull request merged**